roofit/roofitcore/inc/RooAbsPdf.h

```diff
@@ -54,29 +54,31 @@ class RooAbsPdf : public RooAbsReal {
 
   ////////////////////////////////////////////////////////////////////////////////
   /// See RooAbsPdf::generate(const RooArgSet&,const RooCmdArg&,const RooCmdArg&,const RooCmdArg&,const RooCmdArg&,const RooCmdArg&,const RooCmdArg&)
+  /// \param[in] whatVars set
```

Comment:
```suggestion
  /// \param[in] whatVars Set of observables to generate for each event according to this model.
```
Please write parameter descriptions stating with a capital letter that are more than one word